test/test-data-named-groups-unicode.json

```diff
@@ -62,5 +62,27 @@
     "name": "SyntaxError",
     "message": "Invalid escape sequence at position 3\n    (?<\\u{ffffff})\n       ^",
     "input": "(?<\\u{ffffff})"
+  },
+  "(?<$𐒤>a)": {
+    "type": "group",
+    "behavior": "normal",
+    "body": [{
+      "type": "value",
+      "kind": "symbol",
+      "codePoint": 97,
+      "range": [7, 8],
+      "raw": "a"
+    }],
+    "range": [0, 9],
+    "raw": "(?<$𐒤>a)",
+    "name": {
+      "type": "identifier",
+      "value": "$𐒤",
+      "range": [
+        3,
+        6
+      ],
+      "raw": "$𐒤"
+    }
```

Comment:
Maybe add a few more tests? Here are some more examples of differences between ES5 and ES2015+ identifiers: https://mathiasbynens.be/notes/javascript-identifiers-es6